README.md

```diff
@@ -14,6 +14,11 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
   <a href="#join-us">Join us</a>
 </p>
 
+**Carbon is a
```

Comment:
How prominently do we want to advertise Google programs? Maybe move this to #join-us?

Reply:
Just for comparison, I'd noticed Swift has https://www.swift.org/gsoc2023/, but I'm not sure they have links (https://github.com/apple/swift-org-website/pull/225 didn't add any).

Reply:
Done.